custom_metrics/almanac.js

```diff
@@ -278,19 +278,42 @@ return JSON.stringify({
     return parsedNodes;
   })(),
   '12.11': (() => {
+    function containsAnSvg(element) {
+      var children = Array.from(element.childNodes);
+      return !!children.find((child) => {
```

Comment:
Actually could this be simplified to

```js
function containsAnSvg(element) {
  return !!element.querySelector('svg');
}
```

Reply:
Good call. I'll update it in my other branch